src/devhub/entity/community/Discussions.jsx

```diff
@@ -0,0 +1,149 @@
+const { handle } = props;
+const { getCommunity, setCommunitySocialDB } = VM.require(
+  "${REPL_DEVHUB}/widget/core.adapter.devhub-contract"
+);
+
+getCommunity = getCommunity || (() => <></>);
+setCommunitySocialDB = setCommunitySocialDB || (() => <></>);
+
+const communityData = getCommunity({ handle });
+
+const MainContent = styled.div`
+  padding-left: 2rem;
+  flex: 3;
+  @media screen and (max-width: 960px) {
+    padding-left: 0rem;
+  }
+  .post:hover {
+    background-color: inherit !important;
+  }
+`;
+
+const SidebarContainer = styled.div`
+  flex: 1;
+`;
+
+const Heading = styled.div`
+  font-size: 19px;
+  font-weight: 600;
+`;
+
+const SubHeading = styled.div`
+  font-size: 15px;
+  font-weight: 600;
+`;
+
+const Container = styled.div`
+  flex-wrap: no-wrap;
+  max-width: 100%;
+
+  .max-width-100 {
+    max-width: 100%;
+  }
+  @media screen and (max-width: 960px) {
+    flex-wrap: wrap;
+  }
+
+  .card {
+    border-radius: 1rem !important;
+  }
+`;
+
+const Tag = styled.div`
+  border-top-right-radius: 50px;
+  border-bottom-right-radius: 50px;
+  border-top-left-radius: 50px;
+  border-bottom-left-radius: 50px;
+  padding-inline: 0.8rem;
+  padding-block: 0.3rem;
+  display: flex;
+  gap: 0.5rem;
+  border-width: 1px;
+  border-style: solid;
+  font-size: 14px;
+  color: rgba(0, 236, 151, 1);
+  font-weight: 800;
+`;
+
+const [sort, setSort] = useState("timedesc");
+
+return (
+  <div className="w-100" style={{ maxWidth: "100%" }}>
+    <Container className="d-flex gap-3 m-3 pl-2">
+      <MainContent className="max-width-100">
+        <div className="d-flex flex-column gap-4">
+          {context.accountId && (
+            <div className="card p-4">
+              <Widget
+                src={"${REPL_DEVHUB}/widget/devhub.entity.community.Compose"}
+                props={{
+                  onSubmit: (v) =>
+                    createDiscussion({
+                      handle,
+                      data: v,
+                    }),
+                }}
+              />
+            </div>
+          )}
+          <div className="d-flex flex-wrap justify-content-between">
+            <Heading>Announcements</Heading>
+            <div className="d-flex align-items-center gap-2">
+              <select
+                name="sort"
+                id="sort"
+                class="form-select"
+                value={sort}
+                onChange={(e) => {
+                  setSort(e.target.value);
+                }}
+              >
+                <option selected value="timedesc">
+                  Latest
+                </option>
+                <option value="recentcommentdesc">Last Commented</option>
+              </select>
+            </div>
+          </div>
+
+          <Widget
+            src="${REPL_DEVHUB}/widget/devhub.components.organism.Feed"
+            props={{
+              showFlagAccountFeature: true,
+              action: "repost",
```

Comment:
This file is mostly a copy of the `Announcements.jsx` feed only difference is that it looks for `repost` instead of `post`. And the Compose component call createDiscussion in the devhub-contract.